client.go

```diff
@@ -206,6 +206,15 @@ type Client interface {
 	// Get SB_Global table options
 	SBGlobalGetOptions() (map[string]string, error)
 
+	// Creates a new port group in the Port_Group table named "group" with optional "ports" added to the group.
+	PortGroupAdd(group string, ports []string, external_ids map[string]string) (*OvnCommand, error)
+	// Sets "ports" and/or "external_ids" on the port group named "group". It is an error if group does not exist.
+	PortGroupSet(group string, ports []string, external_ids map[string]string) (*OvnCommand, error)
+	// Deletes port group "group". It is an error if "group" does not exist.
+	PortGroupDel(group string) (*OvnCommand, error)
+	// Get PortGroup data structure if it exists
+	PortGroupGet(group string) ([]*PortGroup, error)
```

Comment:
Port_Group's name column is unique (index), so I think it doesn't make sense to return an array. It is better to return a pointer - nil if doesn't exist.

Reply:
Will do.